src/Symfony/Component/String/AbstractUnicodeString.php

```diff
@@ -396,6 +396,21 @@ public function trimStart(string $chars = " \t\n\r\0\x0B\x0C\u{A0}\u{FEFF}"): pa
         return $str;
     }
 
+    public function underscore(): parent
+    {
+        $str = $this->camel()->title();
+        $str->string = mb_strtolower(
```

Comment:
let's make this a oneliner as in `snake()`

Reply:
shouln't this use `$str->lower()` actually ?

Reply:
shouln't this use `$str->lower()` actually ?

Reply:
One liner is IMO too long. 120 characters is good threshold to fit on all screens. But sure can change it.

Reply:
With `lower()` it does not produce underscore between words